src/v2/styles/AttachmentList/AttachmentList-layout.scss

```diff
@@ -52,6 +60,23 @@
     overflow: hidden;
   }
 
+  .str-chat__message-attachment--image .str-chat__image-fallback {
+    @include utils.clamped-height-from-original-image-dimensions(
+                    '--str-chat__image-fallback-max-height',
```

Comment:
I think we should use the same max width and height that we use for regular images (`--str-chat__attachment-max-width`). The reason: if we use different values the image's height can change once the `str-chat__image-fallback` class is applied, which can lead to scrolling issues.

Reply:
We can change the height, but we should use the same height for the fallback as for the regular image.

Reply:
AH, I see, because it's a square, you didn't want to use the default `300px` as the height

Reply:
No, but that could be due to timing issues as well (my guess is that the image error happens AFTER the initial render happened). But items changing sides inside the message list is a known trigger for scroll issues, so I'd avoid them. 

Inspecting the situation further, using a fixed height of `--str-chat__attachment-max-width` (= 300px) isn't 100% either. That's because the original image is displayed with the height computed from `originalHeight`, which could result in an initial height that's different then `--str-chat__attachment-max-width` (for example 200px). And then once the image load fails, we replace the 200px high image with the fallback, that has 300px height. Again, height change happened.

We know, that these things can lead to scroll issues, I think we shouldn't introduce them just because we can't reproduce them at the moment. Down the line it can happen to an integrator, and it will be hard to debug.

Reply:
One more thing to consider: even if we ensure that the original image is the same height as the replacement, if they are displayed in a separate `img` element, what can happen is this:
- we display the original image
- we remove the original image, if the load error happens
- we display the fallback image

I think that technically also counts as a height change within the container, no? I can't tell for 100% that this won't cause scrolling issues.

Reply:
It should not happen, that there is nothing rendered when switching. I would keep objecting that 300px height for fallback is too much - on mobile very bad experience.

Reply:
@szuperaz I have applied the fallback as mask and reverted to 300px default height. I kept the variables specific for the image fallback for better visibility.

Reply:
hm, where do you use the `--str-chat__image-fallback-max-height` variable? I don't see it being used

Reply:
@szuperaz I did not realize I have actually removed them from use. Thank you for spotting this. I have removed those variables.